restli-tools/src/main/java/com/linkedin/restli/tools/compatibility/CompatibilityReport.java

```diff
@@ -55,11 +55,27 @@ public String createReport()
         .map(it -> "[MD-I]:" + it)
         .collect(Collectors.joining("\n"));
 
+    String annotationCompat = "";
+    String annotationIncompat = "";
+    String annotationIsCompat = "";
+    if(_infoMap.getAnnotationInfo(CompatibilityInfo.Level.INCOMPATIBLE).size() > 0 || _infoMap.getAnnotationInfo(CompatibilityInfo.Level.COMPATIBLE).size() > 0)
```

Comment:
Can you also post example error message to this PR ? like your local test?

Otherwise I was expecting to see it from some unit test.

Reply:
Please check the following example:

[checker] [RS-COMPAT]: true

[checker] [MD-COMPAT]: true

[checker] [SCHEMA-ANNOTATION-COMPAT]: false

[checker] [SCHEMA-ANNOTATION-I]:distance: annotation check test, breaks new readers

[checker] [SCHEMA-ANNOTATION-I]:lastName: annotation check test, breaks new readers

**> Task :datarealm-restaurants-models:checkPegasusSchemaSnapshot** FAILED

FAILURE: Build failed with an exception.

- What went wrong:

Execution failed for task ':datarealm-restaurants-models:checkPegasusSchemaSnapshot'.

> There are incompatible changes, find details in /Users/ybi/workspace/datarealm-restaurants_trunk/build/datarealm-restaurants-models/reports/checkPegasusSchema/compatibilityReport.txt

Reply:
I meant Can you add this to the PR descritpion